extensions/wikia/Forum/ForumHooksHelper.class.php

```diff
@@ -1,6 +1,39 @@
 <?php
 
 class ForumHooksHelper {
+
+	static public function discussionNavigationOverride(
+		OutputPage $output, Page $page, Title $title, $user, $request, $wiki
+	) {
+		if ( ForumHelper::isForum() &&
+		     DiscussionsHelper::areForumsArchivedAndDiscussionsEnabled()
+		) {
+			$output->enableClientCache(true);
+			$output->redirect(self::redirectToDiscussions($title));
+		}
+
+		return false;
+	}
+
+	private static function redirectToDiscussions(
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1142)